src/python/grpcio/grpc/_server.py

```diff
@@ -719,22 +727,26 @@ def _start(state):
     _request_call(state)    
     def cleanup_server(timeout):
       if timeout is None:
-        _stop(state, _UNEXPECTED_EXIT_SERVER_GRACE).wait()
+        _stop(state, state.exit_grace, state.exit_handler_grace).wait()
       else:
-        _stop(state, timeout).wait()
+        _stop(state, timeout, True).wait()
```

Comment:
Oops, good catch.  I changed the API when I was working on this and this is a vestige.